### ISSUE TITLE
First run: /about (served by "punch s") vs /about.html (on generated output)

### ISSUE THREAD

**Issue by maniqui** [laktek/punch#69]
Hi.
I've followed the tutorial (a.k.a. "first run"), and I've noticed the following.
On steps 4 user is required to create an About page and then, on step 5, to add a navbar, by   editing `shared.json` and adding:

```
"navbar": [
    { "label": "Home", "href": "/" },
    { "label": "About", "href": "/about" }
]
```

As you can see, the `href` points to `/about`. So far so good, and it works like a charm when serving the website locally with `punch s`.
Then, if we take a peek at the generated output (stuff saved on `/output`), there is a generated `about.html` file. Cool.
But then, if we serve the generated, static website (for example, using Apache), the link on navbar still points to `/about`. If I click it, I get a 404, which is to expect, as the About content is really on `/about.html`.

In other words, the tutorial website works flawlessly when served by Punch, but it breaks on its final, static version.

So, questions is: How is this supposed to be solved?  
- Via some clever URL rewriting? (for example, via .htaccess, if serving static website using Apache)
- Or maybe as solved in [laktek.com/about/](http://www.laktek.com/about/), where  `/about/` seems to be a folder holding an [index.html](http://www.laktek.com/about/index.html)

Thanks.
